assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -411,4 +410,74 @@ describe('ExecutionResults', () => {
       )
     ).toBeTruthy();
   });
+
+  it('should open remediation modal when clicking on checkID', async () => {
+    const {
+      clusterID,
+      hostnames,
+      loading,
+      catalog,
+      executionError,
+      executionStarted,
+      executionResult,
+      checks,
+    } = prepareStateData('completed');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        hostnames={hostnames}
+        catalogLoading={loading}
+        catalog={catalog}
+        executionLoading={loading}
+        executionStarted={executionStarted}
+        executionData={executionResult}
+        executionError={executionError}
+        clusterSelectedChecks={checks}
+      />
+    );
+
+    const { id: checkID, remediation } = catalog[0];
+    const clickableID = screen.getByText(checkID);
+    expect(clickableID.textContent).toBe(checks[0]);
+    let remediationModalElement = screen.queryByText(remediation);
```

Comment:
You don't need this `let` or even the variable.
Simply do: 
`expect(screen.queryByText(remediation)).not.toBeInDocument()`

Reply:
mmm, now I think of this, it might not work this way :sweat_smile: 
and you need it in 2 steps as you have. Give a try hehe

Reply:
Sure, I can remove it. 
For me I just prefer to use named variables because it helps the reader to clarify what is happening.